examples/accessibility/poster.html

```diff
@@ -49,6 +49,7 @@ <h1>&lt;model-viewer&gt; - Accessibility - Poster</h1>
             <h2>Display poster until user interaction</h2>
             <template>
                 <model-viewer id="poster"
+                    controls
```

Comment:
Is this required to transition from poster to a regular view?

Reply:
No, but it was required to test that the controls were immediately usable after the transition